shopfloor/services/location_content_transfer.py

```diff
@@ -542,6 +545,21 @@ def scan_package(self, location_id, package_level_id, barcode):
             move_lines.mapped("picking_id"), message=self.msg_store.barcode_not_found()
         )
 
+    def _set_or_increment_qty_done(self, move_line, qty=False):
+        """Either increments qty_done or sets it depending on the `no_prefill_qty`
+        option value.
+        """
+        qty = self._get_prefill_qty(move_line, qty)
+        if self.work.menu.no_prefill_qty:
+            move_line.qty_done += qty
+        else:
+            move_line.qty_done = qty
+
+    def _get_prefill_qty(self, move_line, qty=False):
```

Comment:
It should not be supported by this scenario as the qty done is always set on all moves at the beginning.
Quantity should be managed when unloading and not when loading, so completely different from what was done on the other scenario.
Should be dropped from this scenario

Reply:
It has been removed